tests/sqla_models_tests.py

```diff
@@ -54,10 +54,11 @@ def test_is_time_druid_time_col(self):
 
         database = Database(database_name="druid_db", sqlalchemy_uri="druid://db")
         tbl = SqlaTable(table_name="druid_tbl", database=database)
-        col = TableColumn(column_name="__time", type="INTEGER", table=tbl)
-        self.assertEqual(col.is_dttm, None)
-        DruidEngineSpec.alter_new_orm_column(col)
-        self.assertEqual(col.is_dttm, True)
+        column_spec = DruidEngineSpec.get_column_spec(column_name="__time")
+        print(">>> {}".format(column_spec))
+        self.assertTrue(column_spec.is_dttm)
+        col = TableColumn(column_name="__time", type=column_spec.sqla_type, table=tbl)
+        self.assertTrue(col)
```

Comment:
What is the purpose of this assertion? Also, the type of the `type` field on `BaseColumn` and the child class `TableColumn` is `String(32)`, i.e. should not be passed a sqla column type.